export/export_test.go

```diff
@@ -0,0 +1,163 @@
+package export
+
+import (
+	"errors"
+	"testing"
+
+	mockenv "github.com/bitrise-io/go-utils/env/mocks"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestProcessRawExportMap(t *testing.T) {
```

Comment:
We could add a test which checks for more complex file patterns (not just wildcard and env).

For example: *_test.txt or something like that